controllers/tab/issueEntry/form/IssueEntryPublicationMetadataForm.inc.php

```diff
@@ -82,7 +84,7 @@ function fetch($request) {
 		// include issue possibilities
 		import('classes.issue.IssueAction');
 		$issueAction = new IssueAction();
-		$templateMgr->assign('issueOptions', $issueAction->getIssueOptions());
+		$templateMgr->assign('issueOptions', $this->getIssueOptions($context));
 
```

Comment:
If $issueAction no longer provides issue options, is the Import/instantiation above it still required?